### DISCUSSION TITLE
Creality firmware V1.3.3.29 confusion

### DISCUSSION THREAD

**Discussion by SteynMarnus** [Guilouz/Creality-Helper-Script-Wiki#614] [Issues]
I wanted to confirm the firmware version numbers. According to the Creality website, V1.3.3.29 is newer than V1.3.3.8, as V1.3.3.29 was released after V1.3.3.8, so dont know if I can use it to run script.
![image](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/19153368/ec752849-bb75-467d-a493-b5ad71ff9949)

**Comment by jaegerschnitzel** [comment]
Yes 1.3.3.29 is supported.

**Reply by SteynMarnus** [reply]
Great, thank you!

**Comment by TheWebMachine** [comment]
v1.3.3.29 is the latest firmware version for the K1 series (K1, K1C, K1 Max) and the script was updated with links for that version at the same time. Each segment of a version number is seen as a whole integer that increments, thereby 1.3.3.29 is higher than 1.3.3.8. (1.3.4.0 would be newer than both of those, as versioning is read from left to right.

https://guilouz.github.io/Creality-Helper-Script-Wiki/changelog/

**Comment by SteynMarnus** [comment]
Solved -  Can use script with 1.3.3.29
